content/app/github/window-helper.js

```diff
@@ -33,6 +33,15 @@ define("content/app/github/window-helper", function(require, exports, module) {
 				return new Box(0, 0, width, height);
 			},
 
+			/**
+			 * Scrolls vertically document body.
+			 * @param  {Number}		scrollDelta
+			 */
+			scrollTop : function(scrollDelta) {
```

Comment:
:+1: 

Great proposal, it is much simpler in that way. However, cases where you try the scroll more then allowed are handled by the browser. So even if you try some extreme value, content will be scrolled to min / max valid value.